packages/Radio/README.md

```diff
@@ -0,0 +1,31 @@
+## Radio
+
+The `<Radio>` component displays a radio input and label text beside it. When clicked it selects the input and deselects any other radio input in its group. It can also have an indeterminate state which can be used to group checkboxes.
+
+### Installation
+
+`> npm install --save @paprika/radio`
+or
+`> yarn add @paprika/radio`
+
+### Usage
+
+```js
+import Radio from "@paprika/radio";
+
+<Radio onChange={handleChange} checkState={checkedStateValue>
```

Comment:
can you add an example of radio group please, I think that one would be the most common use case.